src/mame/drivers/savquest.cpp

```diff
@@ -831,22 +919,27 @@ void savquest_state::savquest(machine_config &config)
 	pcat_common(config);
 	DS12885(config.replace(), "rtc");
 
-	pci_bus_legacy_device &pcibus(PCI_BUS_LEGACY(config, "pcibus", 0, 0));
+	pci_bus_legacy_device &pcibus(PCI_BUS_LEGACY(config, "pcibus", 34_MHz_XTAL, 0)); // PII PCI bus clock 33/34/37/40.1 MHz
 	pcibus.set_device( 0, FUNC(savquest_state::intel82439tx_pci_r), FUNC(savquest_state::intel82439tx_pci_w));
 	pcibus.set_device( 7, FUNC(savquest_state::intel82371ab_pci_r), FUNC(savquest_state::intel82371ab_pci_w));
 	pcibus.set_device(13, FUNC(savquest_state::pci_3dfx_r), FUNC(savquest_state::pci_3dfx_w));
 
-	ide_controller_32_device &ide(IDE_CONTROLLER_32(config, "ide").options(ata_devices, "hdd", nullptr, true));
+
+	ide_controller_32_device &ide(IDE_CONTROLLER_32(config, "ide").options(ata_devices, "hdd", "cdrom", true));
 	ide.irq_handler().set("pic8259_2", FUNC(pic8259_device::ir6_w));
 
-	ide_controller_32_device &ide2(IDE_CONTROLLER_32(config, "ide2").options(ata_devices, nullptr, nullptr, true));
+	ide_controller_32_device &ide2(IDE_CONTROLLER_32(config, "ide2").options(ata_devices, "hdd", nullptr, true));
 	ide2.irq_handler().set("pic8259_2", FUNC(pic8259_device::ir7_w));
 
-	/* sound hardware */
+	/* printer */
+	PC_LPT(config, m_lpt);
+	m_lpt->irq_handler().set("pic8259_2", FUNC(pic8259_device::ir4_w));
```

Comment:
I've fixed the merge conflict because there's something odd on this PR and I want to verify in person via the CI build:
This will override the legacy HASP dongle (cfr. in `savquest_io`) without giving any alternative back.
Current bleeding edge build shows that the Voodoo switches to an odd command prompt debug mode instead, but the "renders incorrectly" comment implies that it actually switches correctly here, implying that there's a debug switch enabled somewhere in the protection?